app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -1970,6 +1976,9 @@ public void setFragmentListener(final PlayerServiceEventListener listener) {
         triggerProgressUpdate();
     }
 
+    /** Remove the listener, if it was set.
+     * @param listener listener to remove
+     * */
```

Comment:
Why do you write Javadocs this way? In my Android Studio when I type `/**` I already get the correctly formatted Javadoc. It's not a big issue, but it's strange you are deviating from the style used throughout the project, especially if it's also the standard style in Android Studio

Reply:
I think it’s because I always started typing after `/**`, and then Android Studio does not complete the arguments automatically. Since I started hitting Enter after `/**` the comment automatically gets created, you are right.